packages/client/src/api/auth/index.ts

```diff
@@ -10,6 +15,18 @@ class AuthApi extends BaseApi {
       data: { ...trimData(payload), ...userStubFields },
     })
 
+  signIn = async (payload: AuthPayload) =>
+    this.createRequest(`${this.baseUrl}/signin`, {
+      shouldParseResponse: false,
+      method: HttpMethod.POST,
+      data: { ...trimData(payload) },
```

Comment:
```suggestion
      data: trimData(payload),
```

Здесь можно не создавать новый объект, а сразу передать результат функции